DQMOffline/Lumi/plugins/ZCounting.cc

```diff
@@ -864,6 +858,20 @@ bool ZCounting::isMuonTriggerObj(const ZCountingTrigger::TTrigger& triggerMenu,
   return false;
 }
 
+//--------------------------------------------------------------------------------------------------
+// Definition of the CustomTightID function
+bool ZCounting::isCustomTightMuon(const reco::Muon& muon) {
+  if (!muon.isPFMuon() || !muon.isGlobalMuon())
+    return false;
+
+  bool muID = isGoodMuon(muon, muon::GlobalMuonPromptTight) && (muon.numberOfMatchedStations() > 1);
+
+  bool hits = muon.innerTrack()->hitPattern().trackerLayersWithMeasurement() > 5 &&
+              muon.innerTrack()->hitPattern().numberOfValidPixelHits() > 0;
+
+  return muID && hits;
```

Comment:
```suggestion
  bool muIdAndHits = muId &&
              muon.innerTrack()->hitPattern().trackerLayersWithMeasurement() > 5 &&
              muon.innerTrack()->hitPattern().numberOfValidPixelHits() > 0;

  return muIdAndHits;
```
can save some computation if muID is already `false`

Reply:
After the modifications, how can I make the squash of all the commits?

Reply:
> After the modifications, how can I make the squash of all the commits?

After made all modifications, and committed, you can:
```
git log
git reset --soft [commit]    # where [commit] is the last commit from the log previous to the first one of yours
git commit -m "any comment"
git push -f my-cmssw [theBranchName]
```

Now I see that you have several merge commits in the list of your commits... That could be a problem (see "Use of CMS commands" in http://cms-sw.github.io/tutorial-resolve-conflicts.html). Then, either forget my request to squash all commits into one, or be prepared to make a new PR from scratch by copying in another commit the modifications you made .